util.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import sys
+from pathlib import Path
+
+LOG_PATH = Path('/app/logs')
```

Comment:
I'm afraid we will have to use
```
log_file = os.path.join(os.path.dirname(__loader__.path), IMPORT_LOG_PATH, f"{city}.log")
```

or something similar here, so that this setup works
1) inside the container
2) outside the container, with a totally different directory inside the repo,
3) when called on the command line in the top directory
4) when called by flask.

Reply:
Otherwise I will get
```
FileNotFoundError: [Errno 2] No such file or directory: '/app/logs/helsinki.log'
```
when running locally.